### ISSUE TITLE
command 'extension.goto-symbol-stack.goToSymbolDeclaration' not found

### ISSUE THREAD

**Issue by dungru** [miconda/vscode-goto-symbol-stack#5]
After I update to the newest vscode, every time I use the option+], it will get the message.
Seems there is no command.

**Comment by miconda** [comment]
Can you check if the extension is still properly installed and not disabled somehow? Try eventually to reinstall it.

**Comment by dungru** [comment]
Yes, it works for local, but fail when I open the server source code by remote ssh.
Seems vscode extension goto-symbol-stack does not work for remote ssh?

**Comment by dungru** [comment]
![image](https://user-images.githubusercontent.com/5992038/74599328-e6877e80-50ba-11ea-89a0-b56e04ffdbbf.png)

**Comment by miconda** [comment]
Can you specify the steps to reproduce? What do you mean by "does not work for remote ssh"? Do you open the file via ssh or mount remote folder via sshfs?

**Comment by dungru** [comment]
I open the file by ssh which uses the vscode extension.
The detail of the setting remote ssh is here. https://code.visualstudio.com/docs/remote/ssh
It could open the server file just like we open the local file.
No need to mount by using sshfs.

The extension will be install both on the server-side and the local-side.
I have tried other extensions such as Microsoft C/C++ extension. It works fine.

**Comment by miconda** [comment]
Unfortunately I am not familiar with `remote/ssh` extension, whether is adding some limitations on parsing files and locating symbols over ssh. I am also not that focused on vscode plugin development.

If someone finds the issue of using this plugin with `remote/ssh` extension, can fix it and make a PR, I will accept it. But at the moment I can't spend any time to investigate what `remote/ssh` extension impacts and how it affects this plugin.
